spec/rspec_puppet_facts_spec.rb

```diff
@@ -855,8 +816,8 @@
             {
               "operatingsystem" => "RedHat",
               "operatingsystemrelease" => [
-                "6",
-                "7",
+                "8",
+                "9"
```

Comment:
Mind keeping the trailing comma?
```suggestion
                "9",
```